src/api/app/models/bs_request.rb

```diff
@@ -52,6 +52,8 @@ class SaveError < APIException
   has_many :request_history_elements, -> { order(:created_at) }, class_name: 'HistoryElement::Request', foreign_key: :op_object_id
   has_many :review_history_elements, through: :reviews, source: :history_elements
 
+  belongs_to :user, foreign_key: 'creator', primary_key: 'login'
```

Comment:
Yes, we should indeed use an id field instead of a string. However, I don't want to change everything now, small improvements step by step ;)

The primary_key is in the user table: ``:login``, the foreign_key is in the bs_requests table ``:creator``